component/admin/views/icalevent/tmpl/overview.php

```diff
@@ -170,9 +170,14 @@
 							}
 							else
 							{
+
+								$firstRepeat = $row->getFirstRepeat();
 								$times = '<table style="border: 1px solid #666666; width:100%;">';
-								$times .= '<tr><td>' . JText::_('JEV_FROM') . ' : ' . ($row->alldayevent() ? JString::substr($row->publish_up(), 0, 10) : JString::substr($row->publish_up(),0,16)) . '</td></tr>';
+								$times .= '<tr><td>' . JText::_('JEV_FROM') . ' : ' . ($firstRepeat->alldayevent() ? JString::substr($firstRepeat->publish_up(), 0, 10) : JString::substr($firstRepeat->publish_up(),0,16)) . '</td></tr>';
```

Comment:
Ideally we should only display this extra row of data IFF $firstRepeat->publish_up() is not the same as $row->publish_up() i.e. if the repeating event starts in the future the extra data is redundant

Reply:
I disagree as people viewing it will want to see both i.e. if next is same as first then it's not already occurring for instance.